ttle.py

```diff
@@ -101,10 +106,48 @@ def __init__(self):
         ShowBase.__init__(self)
         aspect2d.setAntialias(AntialiasAttrib.MAuto)
 
+        # Create the framerate meter
+        flag = self.config.GetBool('show-frame-rate-meter', False)
+        if flag:
+            self.toggleFrameRateMeter(flag)
+
         from toontown.leveleditor import LevelEditor
         self.le = LevelEditor.LevelEditor()
         self.le.startUp(args.dnaPath)
 
+    def setFrameRateMeter(self, flag):
+        return
```

Comment:
Couldn't the toggleFrameRateMeter stuff just be put in here so we don't need a separate function?